src/branch/current.rs

```diff
@@ -0,0 +1,24 @@
+use crossterm::style::Stylize;
+use crate::branch::context::Context;
+use crate::branch::option::Current;
+use crate::connect::Connection;
+
+pub async fn main(
+    options: &Current,
+    context: &Context,
+) -> anyhow::Result<()> {
+    if options.shell {
+        if let Some(branch) = &context.branch {
+            eprintln!("{}", branch);
```

Comment:
```suggestion
            println!("{}", branch);
```

This one is meant to be used as "the output" of the command and not a message.